maps/tether/submaps/om_ships/curashuttle.dm

```diff
@@ -0,0 +1,67 @@
+// Compile in the map for CI testing if we're testing compileability of all the maps
+#if MAP_TEST
+#include "curashuttle.dmm"
+#endif
+
+// Map template for spawning the shuttle!
+/datum/map_template/om_ships/hybrid
+	name = "OM Ship - Curabitur Rescue Shuttle (new Z)"
+	desc = "A small corporate rescue shuttle."
+	mappath = 'curashuttle.dmm'
+	annihilate = TRUE
+
+// The shuttle's area(s)
+/area/shuttle/curabitur/curashuttle/eng
+	name = "\improper Curabitur Rescue - Engineering"
+	icon_state = "shuttle2"
+	requires_power = 1
+
+/area/shuttle/curabitur/curashuttle/med
+	name = "\improper Curabitur Rescue - Medbay"
+	icon_state = "shuttle2"
+	requires_power = 1
+
+/area/shuttle/curabitur/curashuttle/hangar
+	name = "\improper Curabitur Rescue - Hangar"
+	icon_state = "shuttle2"
+	requires_power = 1
+
+/area/shuttle/curabitur/curashuttle/cockpit
+	name = "\improper Curabitur Rescue - Cockpit"
+	icon_state = "shuttle2"
+	requires_power = 1
+
+// The shuttle's 'shuttle' computer
+/obj/machinery/computer/shuttle_control/explore/curashuttle
+	name = "short jump console"
+	shuttle_tag = "Private Vessel"
+	req_one_access = list()
+
+// The 'shuttle'
+/datum/shuttle/autodock/overmap/curashuttle
+	name = "Private Vessel"
```

Comment:
This has to be named something else. Shuttles can't have duplicate names, and this will conflict with the generic shuttle. If they are both spawned, the server will burst into flames.

Reply:
Fixed!